lib/sycamore/sycamore/config.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional, Any
+
+
+class Config:
```

Comment:
Why not just use a regular `@dataclass` for this, or even a Pydantic `BaseModel` subclass (which gives you serialization and validation for free)? I don't know why one would implement this as a key-value pair mapping :-)

Reply:
Haha fair enough, let me update

Reply:
Discussed offline, will start with a typed BaseModel and can expand it to free-formed fields if needed later